hassio/src/dialogs/datadisk/dialog-hassio-datadisk.ts

```diff
@@ -21,8 +21,8 @@ import type { HomeAssistant } from "../../../../src/types";
 import type { HassioDatatiskDialogParams } from "./show-dialog-hassio-datadisk";
 
 const calculateMoveTime = memoizeOne((supervisor: Supervisor): number => {
-  const speed = supervisor.host.disk_life_time !== "" ? 30 : 10;
-  const moveTime = (supervisor.host.disk_used * 1000) / 60 / speed;
+  // Assume a speed of 30 MB/s.
+  const moveTime = (supervisor.host.disk_used * 1000) / 60 / 30;
```

Comment:
It seems this is using the `disk_life_time` estimate as indication of if this is an eMMC (and with that an estimated disk speed).

From what I can tell the Supervisor API always returned `null` if there was no eMMC disk life time used estimate available (see https://github.com/home-assistant/supervisor/pull/2413).

Anyhow, since most folks use faster storage nowadays, 30MB/s seems like a reasonable general estimate today.

So essentially, this code defaulted to 30MB/s anyways.